.github/workflows/build-test.yml

```diff
@@ -5,7 +5,7 @@ name: Build & Test
 # Triggers the workflow on push or pull request events but only for the main branch
 on:
   push:
-    branches: [ main ]
```

Comment:
please revert this

Reply:
@elBoberido I added this temporarily so that I could run ci and check the build

Reply:
@elBoberido fixed